Product

```diff
@@ -0,0 +1,34 @@
+class Product: 
+    def __init__(self, sale_price, number_products, purchase_price, name, brand, image, barcode, instructions, 
+        size):
+        self.sale_price = sale_price # needs to be an integer
+        self.number_products = number_products # needs to be an integer
+        self.purchase_price = purchase_price # needs to be an integer
+        self.name = name # needs to be a string
+        self.brand = brand # needs to be a string 
+        self.image = image # needs to be a string
+        self.barcode = barcode # needs to be boolean 
+        self.instructions = instructions # needs to be a string 
+        self.size = size # needs to be an integer 
+
+
+    def design(): # we will need certain sizes, materials and colors
+        return ('We determine the size, texture and appropiate features the product should have to catch the attention of the users')
+
+        # we should return the appropiate combinations of the factors above 
+
+    def get_price_product(): # we will need to take into account factors like how difficult it was to 
```

Comment:
You're not using Docstrings and this is not a correct structure for python comments.

Reply:
I modified my comments' structure, making use of docstrings for better understanding of what my program actually does.